views/_blocks/recaptcha_view.php

```diff
@@ -0,0 +1,27 @@
+<script src="<?php echo $recaptcha_script_url; ?>" async defer></script>
```

Comment:
This follows https://developers.google.com/recaptcha/docs/faq#does-recaptcha-support-users-that-dont-have-javascript-enabled